Source/WebCore/editing/CompositeEditCommand.cpp

```diff
@@ -959,9 +959,9 @@ void CompositeEditCommand::rebalanceWhitespaceOnTextSubstring(Text& textNode, in
     String string = text.substring(upstream, length);
     // FIXME: Because of the problem mentioned at the top of this function, we must also use nbsps at the start/end of the string because
     // this function doesn't get all surrounding whitespace, just the whitespace in the current text node.
+    const bool nextSiblingIsTextNode = textNode.nextSibling() && textNode.nextSibling()->isTextNode() && downcast<Text>(textNode.nextSibling())->data().length() && downcast<Text>(textNode.nextSibling())->data()[0] != ' ';
```

Comment:
I'd rather call this variable nextSiblingIsTextNodeWithoutLeadingSpace to be more explicit.

Reply:
Fixed but EWS hook is not picking the latest change yet.